src/test/scala/nl/knaw/dans/easy/solr/ConfSpec.scala

```diff
@@ -0,0 +1,107 @@
+package nl.knaw.dans.easy.solr
+
+import java.io.{ByteArrayOutputStream, File}
+import java.lang.System.{clearProperty, setProperty}
+import java.net.MalformedURLException
+
+import org.apache.commons.configuration.PropertiesConfiguration
+import org.apache.commons.io.FileUtils.{readFileToString, write}
+import org.scalatest.{FlatSpec, Matchers}
+
+import scala.collection.JavaConverters._
+import scala.runtime.BoxedUnit
+
+class ConfSpec extends FlatSpec with Matchers {
+
+  // by default tests don't use any cfg/application.properties
+  clearProperty("app.home")
+
+  "options in help info" should "be part of README.md" in {
+    // clearProperty("app.homedir") does not help a maven build so this test should come first
```

Comment:
Last time I tried it looked like clearProperty did not work as expected, but I must have made a... AHA ! Now I see the mistake: app.home versus app.homedir confusion.